m2m/tag_mapper.py

```diff
@@ -24,6 +26,27 @@ def to_markdown(self):
             return self.markdown_unordered_list()
         elif self.tag.name == 'ol':
             return self.markdown_ordered_list()
+        # add div
+        elif self.tag.name == 'div':
+            return self.markdown_div()
+
+    def deal_div(self, div_tag):
+        for child in div_tag:
+            if child.name == 'div':
+                temp = self.deal_div(child)
+                if temp:
+                    self.s += temp
```

Comment:
Does it need to handle div inside div in this case? If not, we could remove this logic, keeping only the link inside div:

```python
def parse_div(self, div_tag):
    for child in div_tag:
        if child.name == 'a' and child.has_attr('href'):
            link_text = reduce(lambda result_text, current_text: self.parse_text(result_text, current_text), child, "")
            return f"[{link_text}]({child['href']})"

        return ''
```

Reply:
Well, maybe in most cases there will be no continuous nesting of `div`, but I think we can keep this. And the `div` doesn't have to just nest `a`, maybe there are other tags, like `img`, etc. :)